nucliadb/nucliadb/tests/integration/tasks/test_tasks.py

```diff
@@ -31,6 +31,17 @@
 
 pytestmark = pytest.mark.asyncio
 
+WORK_DONE = None
```

Comment:
Be careful, tests relying on global variables can't run in parallel. I'd rather prefer to use a fixture instead or something like the tests you're doing below

Reply:
fixed